### PR TITLE
Resolving Issue #1491

### PR DESCRIPTION
**Pull request by dimaslv**

### CODE REVIEW

File: cli/add_datasource.php [hunk 1]
```diff
@@ -103,9 +103,12 @@
 
 /* display_help - displays the usage of the function */
 function display_help () {
-  print "--host-id=id            - The host id\n";
-  print "--data-template-id=id   - The numerical ID of the data template to be added\n";
-  print "--help                  - display this help message\n";
+  echo "Cacti Add Datasource to Host v0.1\n\n";
+  echo "usage: add_datasource.php [--help] --host-id=[ID] --data-template-id=[ID]\n\n";
+  echo "Cacti utility for creating datasources via a command line interface.\n\n";
```
Comment (netniV) [inline cli/add_datasource.php]: I think given the name of the file, this should really be adding not creating.  That aside, This file seems to be OK, I'm not seeing anything else majorly wrong.  @cigamit may have some thoughts.

Comment (cigamit) [reply cli/add_datasource.php]: Remove the --help.  That is always assumed.  Make these changes and I'm good.

**Comment by cigamit** [comment]
Another issue, copyright dates, and we discourage the use of soft tabs.  Set your tab stop to 4 characters and run through these and convert from soft to hard tabs.

File: cli/add_datasource.php (hunk 1 above)
Comment (dimaslv) [reply cli/add_datasource.php]: I'm really not sure what exactly is going on in the part of code which is doing the main job, as it is just copy paste from data_sources.php. So I don't no how and which output should be printed.

**Comment by dimaslv** [comment]
Anything else?

**Comment by netniV** [comment]
You've done a good job so far.  My only other concern is over the SQL usage directly in the CLI code.  If the code is right, it should be a function in one of the lib/ files so that it is not overlooked at a later point and is in a common place.  @cigamit might say otherwise.

I will probably make some further changes once merged as I am working on some common CLI functions.

**Comment by dimaslv** [comment]
Yeap, SQL is bad. But I want to notice, that this particular script is for repairing problem caused by the Cacti itself and making this script shining isn't good time spending at all. Much better would be make Cacti not to break things apart.
I personally consider this script only as a temporary workaround until it is not possible to correctly edit graph templates is Cacti.

By the way, speaking about fixing the problem - first thing that should be implemented in a library is function for adding datasource to host (thing that as been done in the first script). After that upon adding new datatemplate to graph template, this functions should be called for every host which use that graph template, so that new datasource would be added to every host and that datasource would be correctly graphed.

**Pull request merged**